src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
   </React.StrictMode>,
 );
```

Comment:
- [ ] What you did here actually works but it's not the standard way of react library. It'll be good if you have an App.js component, into which you will call the Calculator.js component and then the App.js component is then called inside of the index.js component. But currently, you named your Calculator.js component call inside of index as App and then passed it in index.js. That is not the required way. 
- [ ] Kindly recreate the App.js component, make it a class-based as well, then call the Calculator.js inside this App.js, and in your index.js file, you can then call the App.js component. 👍